CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.34.0
+
+* `FEAT`: add promise polyfill for phantom-js ([`e9d5969f`](https://github.com/bpmn-io/bpmn-js-properties-panel/commit/e9d5969f141e1424a3026fc32f2a380e40507118))
+* `FEAT`: add variable name hint for form field ids ([`f0d04332`](https://github.com/bpmn-io/bpmn-js-properties-panel/commit/f0d0433291e9049062aee17b67a179bb3bcdfeb5))
+* `FEAT`: add tooltip support for dropdown options ([`2aa74ee8`](https://github.com/bpmn-io/bpmn-js-properties-panel/commit/2aa74ee82781260844e27e5f7fffbba6cb02ad5d))
+* `CHORE`: bump to `bpmn-js@7.2.0`

```

Comment:
Should we maybe add a note like this about the peer deps?

https://github.com/bpmn-io/bpmn-js-properties-panel/blob/52afcba696d7ab5b03b082e40779070ec6d772be/CHANGELOG.md#0330

Reply:
BTW peer deps are not updated: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/package.json#L66

Reply:
Alrighty